ui/src/app/pages/profile/data-access-panel.tsx

```diff
@@ -0,0 +1,81 @@
+import {StyledAnchorTag} from 'app/components/buttons';
+import {CheckCircle, ControlledTierBadge} from 'app/components/icons';
+import {styles} from 'app/pages/profile/profile-styles';
+import colors from 'app/styles/colors';
+import * as Utils from 'app/utils';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+const needsAgreementText = 'Contains expanded participant data, including genomics. Before you can access controlled tier data, ' +
+  'your institution will need to sign an amended agreement with the All of Us Data and Research Center.';
+
+const RegisteredTierSection = ({isInRegisteredTier = false}) => {
+  return <div style={{
+    marginBottom: '1rem',
+    display: 'grid',
+    gridTemplateColumns: 'fit-content(10rem) 1fr',
+    gridTemplateAreas: `"regPrimary regAvailable"
+                          "regSecondary regSecondary"`
+  }}>
+    <div style={{...styles.inputLabel, gridArea: 'regPrimary', marginRight: '0.5rem'}}>Registered tier</div>
+    {isInRegisteredTier
+      ? <CheckCircle style={{gridArea: 'regAvailable'}} color={colors.success} size={23}/>
+      : <div style={{ ...styles.dataAccessText, gridArea: 'regSecondary'}}>
+          Please complete data access requirements below to gain access to registered tier data.
+        </div>
+    }
+  </div>;
+};
+
+const ControlledTierSection = ({ hasInstitutionalAgreement = false, isInControlledTier = false, userRevoked = false}) => {
+  return <div style={{
+    marginBottom: '0.9rem',
+    display: 'grid',
+    columnGap: '0.25rem',
+    width: 459,
+    gridTemplateColumns: 'fit-content(2rem) fit-content(10rem) 1fr',
+    gridTemplateAreas: `"ctBadge ctLabel ctAvailable"
+                          ". ctPrimary ctPrimary"
+                          ". ctSecondary ctSecondary"`
+  }}>
+    <ControlledTierBadge style={{gridArea: 'ctBadge'}}/>
+    <div style={{...styles.inputLabel, gridArea: 'ctLabel'}}>Controlled tier</div>
+    {Utils.cond<React.ReactElement>(
```

Comment:
could you give a general guideline for when we'd want to use this `cond` over `fp.cond` ?  Maybe as a comment for cond?

I like how it organizes the logic.